.github/workflows/build-and-test.yml

```diff
@@ -97,6 +97,7 @@ jobs:
           pip install --no-cache-dir --constraint requirements.txt scancode-toolkit==$SCANCODE_VERSION
     - name: Checkout Repository
       uses: actions/checkout@v3
+      submodules: recursive
```

Comment:
This needs to be indented and nested below `with`. See the [usage](https://github.com/actions/checkout#usage).

Reply:
fixed